src/app/modules/myinfo/myinfo.service.ts

```diff
@@ -241,7 +241,7 @@ export class MyInfoService {
    * @returns the person object retrieved.
    * @throws an error on fetch failure or if circuit breaker is in the opened state. Use {@link CircuitBreaker#isOurError} to determine if a rejection was a result of the circuit breaker or the action.
    */
-  fetchMyInfoPersonData(
+  private fetchMyInfoPersonData(
```

Comment:
should we use Javascripts' `#` private modifier instead of TypeScript's `private` modifier?

See this [link](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#ecmascript-private-fields) for difference.

Reply:
noted; shifted `fetchMyInfoPersonData` tests into the tests for `fetchMyInfoData`